### PR TITLE
Features/fanta 18 + fanta 19

### PR DESCRIPTION
**Pull request by jhentai1994**
Check it out

### CODE REVIEW

File: client/src/components/InventoryItem/InventoryItem.scss [hunk 1]
```diff
@@ -75,16 +75,30 @@
...
             @include tablet {
               padding-top: 1.75rem;
             }
           }
           &-delete{
-            height:1rem;
+            position: absolute;
+            top:3.5rem;
+            right:-1.2rem;
+            display: none;
```
Comment (durph) [inline client/src/components/InventoryItem/InventoryItem.scss]: position in mobile width looks a bit off. `top: 1.5rem; right: 0.8rem;` or something like that looks a bit closer to the kebab.

File: client/src/components/Inventory/Inventory.js [hunk 2]
```diff
@@ -31,74 +24,64 @@ class Inventory extends Component {
...
+        for(let e of arr){
+          e.style.display = "none";
+        }
+      }
+      else{
+        let arr = document.getElementsByClassName("inventory__list-entry-item-kebab-delete");
+        for(let e of arr){
+          e.style.display = "none";
+        }
+        let target = e.target.parentNode.firstChild.nextSibling;
```
Comment (durph) [inline client/src/components/Inventory/Inventory.js]: I don't think we should be using document.getElementByWhatever. I think we should try to do things with props and refs etc.
I have an idea but hard to type out. I can share tomorrow.

File: client/src/components/InventoryItem/InventoryItem.scss (hunk 1 above)
Comment (jhentai1994) [reply client/src/components/InventoryItem/InventoryItem.scss]: made changes

File: client/src/components/Inventory/Inventory.js (hunk 2 above)
Comment (jhentai1994) [reply client/src/components/Inventory/Inventory.js]: made changes

**Comment by durph** [comment]
it looks good and all works.

One problem is that now that the warehouse details page is there, the delete button needs to show up on that page as well and delete the item. All the necessary code is written but I think you'll need to change the level at which some of it lives.

InventoryList component is what's included in WarehouseDetails component, so whatever is given to InventoryList component from its parent (Inventory) would need to be given to it in the other context as well.

**Comment by jhentai1994** [comment]
k, on it

**Comment by durph** [comment]
awesome! it all works

**Pull request merged**